File: 240305/시험 통과 여부 확인하기 2/verify-test-passed-2.cpp
#include <iostream>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int n;
    cin >> n;
    int cnt = 0;
    for (int i=1; i<=n; i++){
        int a, b, c, d;
        cin >> a>>b>>c>>d;
        if ((a+b+c+d)/4 >= 60 ){
            cout << "pass";
            cnt += 1;
        }
        else{
            cout << "fail";
        }
        cout << endl;
    }
    cout << cnt;
    return 0;
}

File: 240610/시험 통과 여부 확인하기 2/verify-test-passed-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int cnt = 0;

        for (int ii = 0; ii<n; ii++){
            double mean = 0;
            for (int i = 0; i<4; i++){
                mean += sc.nextInt();
            }

            if (mean/4 >= 60){
                System.out.println("pass");
                cnt++;
            }
            else{
                System.out.println("fail");
            }
        }

        System.out.print(cnt);
    }
}